day_3/story_game.py

```diff
@@ -43,38 +45,71 @@ def infinite_query(phrase, valid_answer):
             answer = input(phrase)
         return answer
 
+    @staticmethod
+    def get_fixed_length_string(string: str, max_length: int) -> str:
+        new_string = ''
+        if len(string) > max_length:
+            space = ' '
+            list_from_string = string.split(space)
+            current_max_length = max_length
+            for word in list_from_string:
+                string_part = word + space
+                if len(new_string) + len(string_part) < current_max_length:
+                    new_string += string_part
+                else:
+                    new_string += '\n' + string_part
+                    current_max_length += max_length
+            new_string.rstrip(space)
+        return new_string if new_string else string
+
+    def text_with_borders_printing(
+            self, tp_brd: str, btm_brd: str,
+            text: str = '',
+            action: Callable = None, action_args: tuple = ()
+    ):
+        result = None
+        brd_length = len(tp_brd.split('\n')[0])
+        print(tp_brd)
+        if text:
+            text_processed = self.get_fixed_length_string(text, max_length=brd_length)
+            print(text_processed)
+        if action:
+            result = action(*action_args)
+        print(btm_brd)
+        return result
```

Comment:
names like `tp_brd`, `btm_brd` and `brd_length` makes it harder to understand what's going on, please avoid them.
It's really hard to understand at glance what's going on in the line: `self, tp_brd: str, btm_brd: str`.